clustered/client/src/main/java/org/ehcache/clustered/client/service/DefaultClusteringService.java

```diff
@@ -86,7 +86,9 @@ public ClusteringServiceConfiguration getConfiguration() {
   @Override
   public void start(final ServiceProvider<Service> serviceProvider) {
     try {
-      clusterConnection = ConnectionFactory.connect(clusterUri, new Properties());
+      Properties properties = new Properties();
+      properties.put("connection.name", "EHCACHE:" + this.entityIdentifier);
```

Comment:
I would *at the very least* use constants for the "EHCACHE" identifier as well as the ":" separator.